docker/statutory/Dockerfile

```diff
@@ -4,8 +4,10 @@ RUN mkdir -p /usr/app/src \
 	&& mkdir -p /usr/app/media \
 	&& mkdir -p /usr/app/scripts
 
-RUN apt-get update
-RUN apt-get install netcat -y
+RUN apt-get update \
```

Comment:
we don't use netcat anymore so you can remove it completely if you want

Reply:
It's still in the wait script, is that also not used anymore?

Reply:
Yeah exactly. I've added it to have a testing setup (probably to wait until the container is alive), but we have pretty much the same with healthchecks now, so it can also be removed